website/common/locales/en/groups.json

```diff
@@ -129,6 +129,8 @@
   "sendGiftHeading": "Send Gift to <%= name %>",
   "sendGiftGemsBalance": "From <%= number %> Gems",
   "sendGiftCost": "Total: $<%= cost %> USD",
+  "sendGiftTotal": "Total:",
+  "sendGiftAmount": "<%= cost %>",
```

Comment:
This is weird, why use an entire translation string that contains only variable data? We don't need to run it through the translation function if nothing's being translated.

Reply:
changed the component, removing line 133.